gg-pac-team3-back/src/app.mjs

```diff
@@ -0,0 +1,30 @@
+import express from 'express';
+import cors from 'cors';
+import favicon from 'express-favicon';
+import logger from 'morgan';
+import userRoutes from './routes/userRoutes.mjs';
+import courseRoutes from './routes/courseRoutes.mjs';
+import mainRouter from './routes/mainRouter.mjs';
+import lessonRoutes from './routes/lessonRoutes.mjs';
+import quizRoutes from './routes/quizRoutes.mjs';
+import userCourseRoutes from './routes/userCourseRoutes.mjs';
+
+const app = express();
+
+// Middleware
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(logger('dev'));
+app.use(express.static('public'));
+app.use(favicon(new URL('./public/favicon.ico', import.meta.url).pathname));
+
+// Routes
+app.use('/', mainRouter);
+app.use('/api/users', userRoutes);
+app.use('/api/courses', courseRoutes);  // Pluralized path
```

Comment:
nit: I would remove these comments. Comments should not be change logs, but should only be used to make the code easier to understand, or to flag future work that needs doing (`// TODO: ` comments)

Reply:
I will delete the comments. Thank you.